### PR TITLE
Feature/episode description

### PR DESCRIPTION
**Pull request by RobinWitt**
related to [fourth User Story](https://github.com/RobinWitt/capstone-project/issues/4)

### CODE REVIEW

File: components/EpisodeDetails/index.js [hunk 1]
```diff
@@ -14,20 +15,51 @@ const StyledArticle = styled.article`
...
   const {
     nummer,
     titel,
     autor,
     hörspielskriptautor,
+    beschreibung,
     veröffentlichungsdatum,
+    kapitel,
     links,
   } = episode;
```
Comment (Long-HaHoang) [inline components/EpisodeDetails/index.js]: I know it's a german application.
Usually we should use english for the naming of variables.
Try using `audioBookSkriptAuthor` instead of `hörspielskriptautor`

Furthermore you can use conventions for naming variables.

File: components/EpisodeDetails/index.js [hunk 2]
```diff
@@ -49,6 +81,62 @@ export default function EpisodeDetails({ episode }) {
       <p>Autor: {autor}</p>
       <p>Hörspielskript-Autor: {hörspielskriptautor}</p>
       <p>Veröffentlichungsdatum: {newDate}</p>
+      {beschreibung ? (
```
Comment (Long-HaHoang) [inline components/EpisodeDetails/index.js]: Just because you can do something, doesn't mean you should, and, that's the case with nested ternary operations. Nesting ternary operators results in the kind of code that may seem clear as day when you write it, but six months later will leave maintainers (or worse - future you) scratching their heads and cursing.

Instead, on the side of clarity, use another line to express the nested operation as a separate statement.

**Review by Long-HaHoang — CHANGES_REQUESTED** [review]
Overall it's functional.
In my honest opinion it's not a clean code, if you nest ternary operators.

File: components/EpisodeDetails/index.js (hunk 1 above)
Comment (RobinWitt) [reply components/EpisodeDetails/index.js]: changed it

File: components/EpisodeDetails/index.js (hunk 2 above)
Comment (RobinWitt) [reply components/EpisodeDetails/index.js]: You are right, I was being lazy there. Both of the functions are now separate components. Thanks for the detailed review and suggestions!

File: pages/episodes/[slug].js [hunk 3]
```diff
@@ -22,7 +22,7 @@ export default function EpisodeDetailsPage() {
     // this logic will change when using remote API
     const [filteredEpisode] = serie
```
Comment (ThomasF85) [inline pages/episodes/[slug].js]: filteredEpisode can be undefined, since slug can be undefined (This happens when you reload the episode details page directly via the browser). This leads to an error in the EpisodeDetails component. You should either make sure that the slug is not empty or use conditional rendering for the usage of the EpisodeDetails component.

File: pages/episodes/[slug].js [hunk 4]
```diff
@@ -22,7 +22,7 @@ export default function EpisodeDetailsPage() {
...
       .slice()
```
Comment (ThomasF85) [inline pages/episodes/[slug].js]: The slice is unnecessary, since filter already returns a new array

File: components/EpisodeDetails/index.js [hunk 5]
```diff
@@ -14,41 +16,79 @@ const StyledArticle = styled.article`
...
-  const splitDate = veröffentlichungsdatum.split("-");
-  const newDate = splitDate[2] + "." + splitDate[1] + "." + splitDate[0];
+  const splitDate = releasedate.split("-");
+  const formattedReleaseDate =
+    splitDate[2] + "." + splitDate[1] + "." + splitDate[0];
 
   return (
     <StyledArticle>
-      <button type="button" onClick={() => router.back()}>
+      <StyledReturnButton type="button" onClick={() => router.back()}>
```
Comment (ThomasF85) [inline components/EpisodeDetails/index.js]: You can shorten this to:
`onClick={router.back}`

**Review by ThomasF85 — CHANGES_REQUESTED** [review]
Looks good, but you have one bug.

File: pages/episodes/[slug].js (hunk 4 above)
Comment (RobinWitt) [reply pages/episodes/[slug].js]: changed it

File: pages/episodes/[slug].js (hunk 3 above)
Comment (RobinWitt) [reply pages/episodes/[slug].js]: fixed it with `if(router.isReady) ...` hope this will do

File: components/EpisodeDetails/index.js (hunk 5 above)
Comment (RobinWitt) [reply components/EpisodeDetails/index.js]: changed it

**Review by SaschaRadojkovic — APPROVED** [review]
i think thats fine
